android/src/main/java/com/docrecog/scan/CameraActivity.java

```diff
@@ -421,6 +422,28 @@ protected void doOnResume() {
         keepScreenOnAwhile();
     }
 
+    public void saveLogFile() {
+            File file = new File(android.os.Environment.getExternalStorageDirectory().getPath()+"/Download", "AccuraEmirates.log");
+            String command = "logcat -f "+ file.getPath()+ " -v time *:V";
+            //command = "mahdi";
+            // + " -v time *:V
+            Log.d(TAG, "command: " + command);
+            String s=android.os.Environment.getExternalStorageDirectory()+"/aaa";
```

Comment:
what is this?

Reply:
did setup for save log files in storage, removed it